### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by jasmainak**
closes https://github.com/mne-tools/mne-bids/issues/29

### CODE REVIEW

**Comment by jasmainak** [comment]
@agramfort can you enable travis? It seems I can't do it

**Comment by agramfort** [comment]
done

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+before_install:
+  - wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b -p /home/travis/miniconda
+  - export PATH=/home/travis/miniconda/bin:$PATH
+  - conda update --yes --quiet conda
+install:
+    - conda create -n testenv --yes pip python=2.7
+    - source activate testenv
+    - conda install --yes --quiet numpy=1.11 scipy=0.17 matplotlib=1.5
```
Comment (teonbrooks) [inline .travis.yml]: we should include pandas here as well

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - conda install --yes --quiet nose coverage
+    - pip install -q flake8 mne
```
Comment (agramfort) [inline .travis.yml]: mne -> mne_bids

**Comment by jasmainak** [comment]
Thanks for the reviews @agramfort and @teonbrooks

**Pull request merged**